force-app/main/default/lwc/link/link.js

```diff
@@ -9,31 +9,32 @@ export default class Link extends LightningElement {
     @track currentPath;
     @track unsubscribe;
     parentUrl
-    async connectedCallback(){
-        await getRouteMatch(this, ({path, url}) => {
+    async connectedCallback() {
+        await getRouteMatch(this, ({ path, url }) => {
             this.parentUrl = url;
-            if(this.to.indexOf(':url') > -1){
+            if (this.to.indexOf(':url') > -1) {
                 this.to = this.to.replace(':url', this.parentUrl);
             }
         })
         await dispatchEvent(REGISTER_ROUTER_EVENT_NAME, this, async (routerInstance) => {
             this.routerInstance = routerInstance;
             this.currentPath = this.routerInstance.currentPath;
-            this.unsubscribe = this.routerInstance.subscribe(this,this.setCurrentPath.bind(this))
+            this.unsubscribe = this.routerInstance.subscribe(this, this.setCurrentPath.bind(this))
         })
     }
-    setCurrentPath (){
+    setCurrentPath() {
         this.currentPath = this.routerInstance.currentPath;
     }
-    handleClick(){
-        if(this.to){
+    handleClick(e) {
```

Comment:
We are using this method in the mainMenuLink (CustomLink example). Could you please also pass the event in `super.handleClick();`. Mean while I will update the documentation. 
https://github.com/chandrakiran-dev/lwc-router/blob/master/examples/lwc/customLink/lwc/mainMenuLink/mainMenuLink.js

Thank you :)

Reply:
Good call, add that change to the commit.